_config.yml

```diff
@@ -115,20 +115,22 @@ partnersC4sponsorsButton: "Become a sponsor"
 
 # Footer
 socialLinks:
- - {link: "https://plus.google.com/b/102444623953913144164", icon: "google-plus"}
- - {link: "https://twitter.com/intent/user?screen_name=GDGLviv", icon: "twitter"}
- - {link: "https://facebook.com/GDGLviv", icon: "facebook"}
- - {permalink: "/feed.xml", icon: "rss"}
+ - {link: "https://www.instagram.com/blockchaincolumbia/", icon: "linkedin"}
```

Comment:
Icon should be Instagram...hmmm it looks like the `sprites/sprites.svg` doesn't have `icon-instagram`. If we find an svg of the Instagram logo in the same style, we can add it in. I suppose this is okay for now?